pennylane/ops/qubit.py

```diff
@@ -2961,6 +2961,73 @@ def diagonalizing_gates(self):
         return [QubitUnitary(self.eigendecomposition["eigvec"].conj().T, wires=list(self.wires))]
 
 
+class Projector(Observable):
+    r"""Projector(basis_state, wires)
+    The basis state observable :math:`P=\ket{i}\bra{i}`.
+
+    The expectation of this observable returns the value
+
+    .. math::
+        \braket{P} = |\braketT{\psi}{\i}|^{2}
+
+    corresponding to the probability of measuring the quantum state in the :math:`i` -th eigenstate.
+
+    **Details:**
+
+    * Number of wires: Any
+    * Number of parameters: 1
+    * Gradient recipe: None
+
+    Args:
+        basis_state (tensor-like): binary input of shape ``(n, )``
+        wires (Iterable): wires that the template acts on
```

Comment:
nitpicking here, but maybe one could relate `i` to `basis_state`, i.e. that `i` is the integer representation of the bit sequence in `basis_state`, which in turn represents a computational basis state? I always find with these things an example helps, like "for example, the projector |11><11|, or in integer notation |3><3|, is created by `basis_state = [1, 1]`.

Reply:
Thanks for the suggestion, @mariaschuld 
I have now added a line in the description as the following:
For example, the projector :math:`\ket{11}\bra{11}` , or in integer notation :math:`\ket{3}\bra{3}` is created by ```basis_state=np.array([1, 1])```.

Btw, I see that  I could run ```make docs``` to check the compilation, but is there a way to preview how the description will look on the web documentation?

Reply:
Please ignore my question, I find the compiled version in ```doc/_build/html/```!